cumulus_library/template_sql/column_datatype.sql.jinja

```diff
@@ -1,8 +1,13 @@
 SELECT
+    {%- if include_table_names %}
+    table_name,
+    {%- endif %}
```

Comment:
Could you unconditionally include it, if you did it last? (i.e. to not break any `[0]` references)

Reply:
i could - i was mirroring the information_schema order here, but that's not the most critical thing to support, I suppose